src/main/java/edu/wisc/my/portlets/feedback/web/FeedbackFormController.java

```diff
@@ -56,9 +61,13 @@
  * @version $Header: /apps/my/CVS/portlets/FeedbackPortlet/src/main/java/edu/wisc/my/portlets/feedback/web/FeedbackFormController.java,v 1.3 2014/10/31 18:32:48 vertein Exp $
  */
 public class FeedbackFormController extends SimpleFormController {
-
+	private Log logger = LogFactory.getLog(this.getClass());
```

Comment:
Is there a suitable inherited logger in the `SimpleFormController` or its parent classes?